pkg/networkaware/networkoverhead/networkoverhead.go

```diff
@@ -197,24 +199,29 @@ func (no *NetworkOverhead) PreFilter(ctx context.Context, state *framework.Cycle
 		return nil, framework.NewStatus(framework.Success, "Pod has no dependencies, return")
 	}
 
-	// Get pods from lister
-	selector := labels.Set(map[string]string{agv1alpha1.AppGroupLabel: agName}).AsSelector()
-	pods, err := no.podLister.List(selector)
```

Comment:
the previous podLister is inherited from scheduler framework's handle, and it filtered out pods that are in terminal state. So this PR would change the behavior to return all pods. so @jpedro1992 please confirm if it breaks the expection.

Reply:
Hi @Huang-Wei and @zwpaper, 

Indeed it would break the expected behavior. Pods in terminal state should not be considered in the plugin. Is it possible to add the previous filter here, remove the terminal state pods? Then, everything should work fine.

Reply:
Definitely, this part should be reverted.